Source/Core/ClippingPlaneCollection.js

```diff
@@ -189,9 +306,9 @@ define([
     }
 
     /**
-     * Checks whether this collection contains the given plane.
+     * Checks whether this collection contains a plane equal to the given plane.
      *
-     * @param {Plane} [plane] The plane to check for.
+     * @param {Plane} [plane] The plane or ClippingPlane to check for.
```

Comment:
@ggetz was there a reason for this to be numerical checks instead of reference checks?

I guess it kind of makes sense though if a user is looking to not add a numerically pre-existing clipping plane.

Reply:
Follows what we do for other collections. The numerical check is used in the `remove` function below as well, but other than that, there is not specific need.